pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -103,6 +103,7 @@ let
     optionalString
     ;
   inherit (lib.trivial) flip;
+  sysEquals = lib.systems.equals;
```

Comment:
I wouldn't create a new variable here for a two times usage.

Reply:
That's a subjective style nit. I'd appreciate it if it wasn't phrased as an instruction to change or an implication of bad code. :wink: 

The existing code in this file is inheriting almost all functions it uses from `lib`. `inherit (lib.systems) equals` would be in keeping with the code, however `equals` is an ambiguous name, so I went with `sysEquals`.

It's less about how many times its used and more about keeping the line length short & readable later on in those two usages.

That said, I have no strong feelings here, I'm happy to change if people prefer a different style.